tests/test_utils.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import os
+sys.path.append(os.path.dirpath(os.path.realpath(__file__)) + '/../')
```

Comment:
Can we avoid this? Maybe we can restructure the project like the first example at https://docs.pytest.org/en/latest/goodpractices.html#tests-outside-application-code.

Reply:
The first example's directory structure looks good. If we rearrange our project like that, will we able to import the project modules directly? like
```python
from autowebcompat import utils
```

Reply:
I think so, if you execute `python3 -m pytest` or `PYTHONPATH=. pytest`.

Instead of restructuring, we can also run `PYTHONPATH=.. pytest`.

Reply:
I think restructuring will also help us in the long run. So better to just do it now.

Reply:
What do you think?

Reply:
I'm not totally sure because the source files are executable scripts rather than modules that you import.
Only `network.py` and `utils.py` are actually modules.
Maybe we can restructure putting `network.py` and `utils.py` in a `autowebcompat` directory, but keeping the other scripts where they are.